### DISCUSSION TITLE
Extended descriptions on Achievement pages

### DISCUSSION THREAD

**Discussion by julkasalsa** [RetroAchievements/RAWeb#709] [Ideas]
An option to add more detailed description to be displayed on each achievement page (game page would remain having standard, short version). Current character limit is enough for most of achievements, but for more complex, or hard to explain needs more space to make them clear. This should be only optional, but it would be better to extend the overlay functionality to include it.

**Comment by luchaos** [comment]
I was thinking of this as well when i saw this achievement in particular - and similar ones with a "see achievement comments for details" description:
http://retroachievements.org/Achievement/52271

It has elaborate descriptions in additional comments which are primary information on the achievement itself.
Introducing an additional field in the database for that purpose sounds good.
When displayed in the overlay as well it should probably be just text only with line-breaks, no formatting. 
Or at least strip the formatting for the api response to look reasonable in the overlay - formatting could still be applied on the site if required.

Update: we could make this part of a "guide" section as well, optionally hidden for people who like to find out by themselves, "no spoilers".
